ada/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    url = models.URLField(
+        max_length=255,
+        blank=True,
+        null=True,
+        default="https://hub.momconnect.za/confirmredirect",
+    )
+    content = models.TextField(
+        default="This entry has no copy",
+        help_text="The content of the mesage that this link was sent in",
+    )
+    symptom_check_url = models.URLField(
+        max_length=200, null=False, blank=False, default="http://symptomcheck.co.za"
```

Comment:
Do we want to allow this field to be blank or null? Surely we would always want to redirect somewhere?